_includes/markdown/Vue.md

```diff
@@ -0,0 +1,110 @@
+
+### Script Element
+The standard way to include Vue in your application is by using a `<script>` tag in your HTML/PHP file just like any other JavaScript file. The Vue documentation recommends using the [official CDN](https://vuejs.org/v2/guide/installation.html#CDN) to make sure your Vue version is always up to date. 
+
+This is the easiest way to get started and perfect for learning the framework. This is a great option if you’re building within a WordPress environment and only have a few small components to create. You may also consider this method if Vue is not the primary framework used on the project. As with any third party JavaScript inclusion, be sure to evaluate the cost/benefit of using the library/framework before starting.
+
+If you are using Vue for isolated components within a single WordPress template be sure to enqueue the script as you normally would for only that instance as to not load Vue where you don’t need it. If you are using Vue throughout a project it is best to concatenate it alongside all your other JavaScript files.
+
+### Including with NPM
+
+[NPM](https://vuejs.org/v2/guide/installation.html#NPM) is the recommended installation method when building large scale applications with Vue. You can tweak your settings to either use Webpack or Browserify for module bundling. This method of installation is recommended for Vue-based applications when you want more custom and fine-grained control over the dependencies being used. You might also consider using this method when a project is taking advantage of NPM as a front-end package manager.
+
+### Installing with a CLI
+
+The [Vue CLI](https://vuejs.org/v2/guide/installation.html#CLI) provides us a quick way to setup and scaffold a Vue project. It makes use of the modern frontend workflows and out of the box, vue-CLI provides hot reloading, minification, asset management, module bundling, linting and a development server to test your application under realistic circumstances. It works with zero configuration from your part as everything is already set up in the [vue-CLI templates](https://github.com/vuejs-templates/). 
+
+The CLI caters from the most simple app ([the simple template](https://github.com/vuejs-templates/simple)), to the most complex one ([Webpack](https://github.com/vuejs-templates/webpack) or [Browserify](https://github.com/vuejs-templates/browserify) template). This is the recommended way to install Vue when you’re building a single page application that runs off an API and will be using the Vue framework exclusively. The CLI will likely need to be used and implemented at the beginning of a project.
```

Comment:
I think we need to be more clear that this wouldn't be a WordPress theme.

Reply:
sure

Reply:
Added a note about this not being a WP theme